permissions/getters.py

```diff
@@ -46,6 +41,4 @@ def get_service_sectors_with_permission(user: AnyUser, *, permission: str) -> mo
     if has_general_permission(user, permission):
         return ServiceSector.objects.all().values("pk")
 
-    service_sector_ids = [pk for pk, perms in user.service_sector_permissions.items() if permission in perms]
-
-    return ServiceSector.objects.filter(models.Q(id__in=service_sector_ids)).values("pk")
+    return ServiceSector.objects.none()
```

Comment:
`get_service_sectors_with_permission` can be dropped if `ReservationFilterSet.get_only_with_handling_permission` was the only place which used it.